rust/tool/Cargo.toml

```
@@ -3,17 +3,11 @@ name = "lanzaboote_tool"
 version = "0.3.0"
 edition = "2021"
 
-[[bin]]
-name = "lzbt"
-path = "src/main.rs"
-
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 anyhow = "1.0.71"
-clap = { version = "4.3.1", features = ["derive"] }
 goblin = "0.6.1"
-nix = { version = "0.26.2", default-features = false, features = [ "fs" ] }
 serde = { version = "1.0.163", features = ["derive"] }
```

If we turn this into a library we should make the constraints less concrete.

```suggestion
serde = { version = "1", features = ["derive"] }
```